docs/api/qiskit-ibm-runtime/qiskit_ibm_runtime.RuntimeEncoder.md

```diff
@@ -34,31 +34,27 @@ If specified, default is a function that gets called for objects that can’t ot
 
 ## Attributes
 
-<span id="runtimeencoder-item-separator" />
+<span id="qiskit_ibm_runtime.RuntimeEncoder.item_separator" />
 
 ### item\_separator
 
-<span id="qiskit_ibm_runtime.RuntimeEncoder.item_separator" />
-
 `= ', '`
 
-<span id="runtimeencoder-key-separator" />
+<span id="qiskit_ibm_runtime.RuntimeEncoder.key_separator" />
 
-### key\_separator
+### key\_separator = '
 
-<span id="qiskit_ibm_runtime.RuntimeEncoder.key_separator" />
+`= ':`
 
 `= ': '`
```

Comment:
This is a regression.

Now:

<img width="272" alt="Screenshot 2023-10-20 at 5 19 17 PM" src="https://github.com/Qiskit/documentation/assets/14852634/8423b351-3b3c-4f74-82cf-803aa793b8b9">

Before:

<img width="174" alt="Screenshot 2023-10-20 at 5 19 30 PM" src="https://github.com/Qiskit/documentation/assets/14852634/5e58cd21-21af-441c-9bb4-9093b9ed887c">

qiskit.org:

<img width="292" alt="Screenshot 2023-10-20 at 5 19 57 PM" src="https://github.com/Qiskit/documentation/assets/14852634/8019992c-3a63-487f-acff-4a2f081a27fc">

Reply:
yeah this seems like a regression, but honestly the "before" layout also doesn't look great, I don't think the `=` should be there at all if it's on the second line

Reply:
Tracked by https://github.com/Qiskit/documentation/issues/224